packages/design-system/src/components/OcDrop/OcDrop.vue

```diff
@@ -138,7 +138,8 @@ export default defineComponent({
     }
 
     const onFocusOut = (event: FocusEvent) => {
-      const focusLeft = event.relatedTarget && !unref(drop).contains(event.relatedTarget as Node)
+      const tippyBox = unref(drop).closest('.tippy-box')
```

Comment:
focusOut event was triggered before the click event and was not bubbling the event up when the hide function was triggered. 
In Chrome/FF the related event was the a tag or button itself, on Safari it was the tippy box. 

Everything is still working fine